tests/app/test_jinja_filters.py

```diff
@@ -442,3 +443,60 @@ def test_other_config_visibility(
 
     other = OtherConfig(Mock(), answer_schema_textfield)
     assert other.open is expected_visibility
+
+
+@patch("app.jinja_filters.flask_babel.get_locale", Mock(return_value="en_GB"))
+def test_single_calculated_summary_question():
```

Comment:
I am not sure what the single in the name refers to, is it needed?

Reply:
Renamed it to just `test_calculated_summary_question`

Reply:
The test name and assertions has changed so this is resolved now.